src/client/public/sign-for-beta.js

```diff
@@ -0,0 +1,149 @@
+/*
+	This is for the styling of the form in the page
```

Comment:
I think it's OK but you should consider the changes in the HTML as I wrote above.

Reply:
I will, this file is per page...